package.json

```diff
@@ -62,6 +63,7 @@
     "prettier": "^1.14.2",
     "rimraf": "^2.6.2",
     "sass-loader": "^7.1.0",
+    "sign-addon": "^2.0.5",
```

Comment:
Could you help me understand why we need this? 😃

Reply:
you really don't -- you obviously sign your addon differently already; I really wanted to use this dark theme, so had to make a signed version of your addon just for myself, and this official npm module provides a mechanism for doing so. I didn't check in the script I used to generate the .xpi (it's really rough, but worked well enough for me). You could just remove this hunk